src/gps/GPS.cpp

```diff
@@ -178,20 +178,20 @@ GPS_RESPONSE GPS::getACK(const char *message, uint32_t waitMillis)
             if ((bytesRead == 767) || (b == '\r')) {
                 if (strnstr((char *)buffer, message, bytesRead) != nullptr) {
 #ifdef GPS_DEBUG
-                    LOG_DEBUG("\r\nFound: %s\r\n", message); // Log the found message
```

Comment:
Indeed, there are more places where there is no new line on purpose, e.g. in the TraceRoute module:
https://github.com/meshtastic/firmware/blob/1f2d972e18d5d954342efb12da3c04d348a042ed/src/modules/TraceRouteModule.cpp#L112

Reply:
This is why it's a draft. This Multiline or 'Salami Logging' was thrown out once when i added rsyslog support. Also it doesn't work with BLE / Protobuf based logging. A few spots have since crept back in. We need to find them and rewrite them to use printf templating.

I am tempted to throw out the 'continuation line' logic in RedirectablePrint to keep those from popping up.

Reply:
Correction - i removed that already ;-)

Reply:
Ayesir, just pointing out such places. Very much in support of this in general :)